install/configuring_inventory_file.adoc

```diff
@@ -1980,8 +1980,23 @@ xref:../install_config/web_console_customization.adoc#install-config-web-console
 
 |===
 
+[[configuring-the-admin-console]]
 == Configuring the Admin Console
 
+[IMPORTANT]
+====
+The Admin Console is currently a Technology Preview feature only.
```

Comment:
The admin console is not tech preview afaik. It's fully supported.

@jwforres @robszumski can you confirm?

Reply:
I'm not aware of a reason it would need to be considered tech preview, but I'll leave that to PM to make the call.

Reply:
@spadgett @jwforres I see in the tech preview tracker spreadsheet from PM, that it's defined as GA, so I will update that. Thanks!

Reply:
Yes, correct, it's GA